adapters/network.js

```diff
@@ -196,7 +196,11 @@ const proxyRequest = async request => {
     case "FORM":
```

Comment:
Don't we need a similar fix for Ln 193?

Reply:
Checked the destinations which use `XML` the response seemed to have xml formatted string which I think is not a problem. Let me know if you think otherwise